### ISSUE TITLE
New firmware version of HDHR changed >none< text as >not in use<

### ISSUE THREAD

**Issue by mmilci** [zynine-/HDHRViewerV2.bundle#68]
New firmware version of HDHR changed >none< text as >not in use<

https://github.com/zynine-/HDHRViewerV2.bundle/blob/21d7fd308a29ea847756e5b5e567a3b82b8c858f/Contents/Code/__init__.py#L832

**Comment by krbvroc1** [comment]
Can confirm - just updated HDHR Prime firmware 

```
Model: HDHR3-CC
Firmware: 20180817 
```

![clipboard01](https://user-images.githubusercontent.com/1335118/44623787-8a1d1780-a8a6-11e8-9dfd-1a8603d5d837.jpg)

**Comment by mmilci** [comment]
Yup I’ve changed the code on my client (added new one so it checks bit none and not in use) and after it continue to work without any issue
